plugins/main/public/react-services/saved-objects.js

```diff
@@ -138,6 +138,90 @@ export class SavedObject {
     }
   }
 
+  /**
+   * Given a dashboard saved object ID, checks if it exists
+   * Returns an object with status information when found, otherwise returns the error message.
+   */
+  static async existsDashboard(dashboardID) {
+    try {
+      const dashboardData = await GenericRequest.request(
+        'GET',
+        `/api/saved_objects/dashboard/${dashboardID}?fields=title`,
+      );
+
+      const title = (((dashboardData || {}).data || {}).attributes || {}).title;
+      const id = ((dashboardData || {}).data || {}).id;
+
+      if (title) {
+        return {
+          data: 'Dashboard found',
+          status: true,
+          statusCode: 200,
+          title,
+          id,
+        };
+      }
+    } catch (error) {
+      // Keep consistency with existsIndexPattern by returning the error message
+      return ((error || {}).data || {}).message || false
+        ? error.data.message
+        : error.message || error;
+    }
+  }
+
+  // Fetch a dashboard saved object by id using _find and filter by id client-side
+  static async getDashboardById(dashboardID) {
```

Comment:
I believe there is a function to search for savedObjects provided by opensearch.

https://github.com/wazuh/wazuh-dashboard-plugins/blob/main/plugins/main/public/react-services/check-index.ts#L18

Reply:
Fixed on https://github.com/wazuh/wazuh-dashboard-plugins/pull/7842/commits/8c9e9a4828b57ab1a46fa855df28169cd6de531c